runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1246,15 +1283,15 @@ func s:HandleCursor(msg)
 
       let curwinid = win_getid(winnr())
       if win_gotoid(s:asmwin)
-        let lnum = search('^' . s:asm_addr)
-        if lnum == 0
-          call s:SendCommand('disassemble $pc')
-        else
-          exe 'sign unplace ' . s:asm_id
-          exe 'sign place ' . s:asm_id . ' line=' . lnum . ' name=debugPC'
-        endif
-
-        call win_gotoid(curwinid)
+	let lnum = search('^' . s:asm_addr)
```

Comment:
there's a few more hunks in the patch like this that changed idents to tabs for some reason :shrug:

Reply:
yep, no idea why. I manually changed some back but stopped at some point because I decided that'd be unnecessary churn.

`:retab` yay or nay?